webrtc.html

```diff
@@ -8284,74 +8284,107 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                   <code><a>RTCDataChannel</a></code> object.</p>
                 </li>
                 <li>
-                  <p>Initialize <var>channel</var>'s <code><a data-for=
-                  "RTCDataChannel">label</a></code> attribute to the value of
-                  the first argument.</p>
+                  <p>Let <var>channel</var> have an [[<dfn id=
```

Comment:
Will fix typo. I agree that we should try to be consistent on the naming and the "link names", if we need links to all slots.